### PR TITLE
[FIX] Job form text fields converted to dropdowns

### PR DESCRIPTION
**Pull request by Prozak8**
https://www.pivotaltracker.com/story/show/161807374

- free text fields converted to dropdowns
- Added 'new' method back to controller
- 'new' method holds arrays for all dropdown lists
- To avoid technical debt, we have added models for CareTypes, Professions & Departments 
- Seed file updated for new models
- View dropdowns now call on new models

### CODE REVIEW

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -15,6 +15,14 @@ def create
     end
   end
 
+  def new
+    @professions = [["Doctor"], ["Nurse"], ["Assistant Nurse"]]
+    @departments = [["Psychiatric"], ["Surgery"]]
+    @caretypes = [["Hospital"], ["Primarycare"], ["Homecare"], ["Geriatric"], ["Other"]]
```
Comment (tochman) [inline app/controllers/jobs_controller.rb]: Please consider the snake case format here `@care_types` ?

File: app/views/jobs/_new.html.haml [hunk 2]
```diff
@@ -7,19 +7,19 @@
...
+            = form.select :profession, options_for_select(@professions), { include_blank: true }, class: "form-input-field"
             = form.label :hospital, class: "form-label"
-            = form.select :hospital, Hospital.all.collect {|c| [ c.name ] }, {include_blank: 'Please select'}, id: "Hospital", class: "form-input-field"
+            = form.select :hospital, Hospital.all.collect {|c| [ c.name ] }, { include_blank: true }, id: "Hospital", class: "form-input-field"
             = form.label :department, class: "form-label"
-            = form.text_field :department, placeholder: "Department", class: "form-input-field"
+            = form.select :department, options_for_select(@departments), { include_blank: true }, class: "form-input-field"
             = form.label :license, class: "form-label"
-            = form.select :license, License.all.collect {|c| [ c.name ] }, {include_blank: 'Please select'}, class: "form-input-field"
+            = form.select :license, License.all.collect {|c| [ c.name ] }, { include_blank: true }, class: "form-input-field"
```
Comment (tochman) [inline app/views/jobs/_new.html.haml:16]: Is there an association between Job and License? A joint table?

File: app/controllers/jobs_controller.rb (hunk 1 above)
Comment (Prozak8) [reply app/controllers/jobs_controller.rb]: Never thought of that. Will do the same for working_hours

File: app/views/jobs/_new.html.haml (hunk 2 above)
Comment (Prozak8) [reply app/views/jobs/_new.html.haml:16]: Not yet although one job should have one or many

File: features/step_definitions/new_object_steps.rb [hunk 3]
```diff
@@ -38,3 +38,21 @@
     FactoryBot.create(:license, license)
     end
 end
+
+Given("the following profession exists") do |table|
+    table.hashes.each do |profession|
+    FactoryBot.create(:profession, profession)
```
Comment (rupertlion) [inline features/step_definitions/new_object_steps.rb]: can we just use create (ie with without factorybot)

**Review by angiecstn — APPROVED** [review]

**Review by oliverochman — APPROVED** [review]
LGMT

**Review by Noelryn — APPROVED** [review]

**Review by angiecstn — APPROVED** [review]

**Pull request merged**